web/react/components/view_image.jsx

```diff
@@ -109,6 +109,22 @@ module.exports = React.createClass({
             }
         );
 
+        $(this.refs.previewArrowLeft.getDOMNode()).hover(
```

Comment:
Looking back over this now that this pull request was reverted, I'm pretty certain it's this chunk that caused the UI to break. The arrows only exist if there is more than one file attached to the post so this.refs.previewArrowLeft/this.refs.previewArrowRight would be undefined when a post only had one attachment.